__tests__/src/services/database/index.js

```diff
@@ -0,0 +1,40 @@
+import db, { factory } from '../../../../src/services/database/index';
+
+const randomDbPath = () => `database/${Math.random()}`;
+const Realm = require('realm');
```

Comment:
Can we use import here instead of require? It gives more consistency with other project parts.

Reply:
done